backend/src/bundles/auth/helpers/token/generate-token/generate-token.ts

```diff
@@ -0,0 +1,27 @@
+import jwt from 'jsonwebtoken';
+
+import { config } from '~/common/config/config.js';
+
+const generateToken = <T extends object>(
```

Comment:
It would be better to have two different functions, each for a different kind of token

Reply:
Sure, I was thinking about this before, but now - changed :)